src/utils/childProcess.ts

```diff
@@ -227,6 +237,12 @@ export async function spawnFile(
       }
       const error: SpawnError = new Error(message);
 
+      Object.defineProperties(error, {
+        [ErrorCommand]: {
+          enumerable: false,
+          value:      `${ command } ${ finalArgs.join(' ') }`,
+        }
+      });
```

Comment:
I'm doing it like this instead of `Object.create(error, { [ErrorCommand]: … ` so that this is not enumerable (that is, doesn't show up when we just print the error).